spotify/models/user.py

```diff
@@ -1,32 +1,25 @@
 """Source implementation for a spotify User"""
 
-import asyncio
 import functools
 from base64 import b64encode
 from typing import (
-    Optional,
     Dict,
-    Union,
     List,
-    Tuple,
     Type,
     TypeVar,
     Union,
-    Coroutine,
     TYPE_CHECKING,
 )
 
-from ..utils import to_id
-from ..http import HTTPUserClient
 from . import URIBase, Image, Device, Context, Player, Playlist, Track, Artist, Library
+from ..http import HTTPUserClient
```

Comment:
These imports got updated when I did Ctrl + alt + o in PyCharm.  I assume that's fine.